server/publications/collections/accounts.js

```diff
@@ -45,6 +45,22 @@ Meteor.publish("Accounts", function (userId) {
 });
 
 /**
+ * Single account
+ * @params {String} userId -  id of user to find
+ */
+Meteor.publish("Accounts.single", function (userId) {
```

Comment:
We don't use a dot naming convention in any other publication, question if we should.